build/Microsoft.Toolkit.Uwp.Input.GazeInteraction.nuspec

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Microsoft.Toolkit.Uwp.Input.GazeInteraction</id>
+    <version>$version$</version>
+    <title>UWP Community Toolkit Eye Gaze Library</title>
+    <authors>Microsoft</authors>
+    <owners>microsoft, nugetuwptoolkit</owners>
+    <iconUrl>https://raw.githubusercontent.com/Microsoft/UWPCommunityToolkit/master/build/nuget.png</iconUrl>
+    <licenseUrl>https://github.com/Microsoft/UWPCommunityToolkit/blob/master/license.md</licenseUrl>
+    <projectUrl>https://github.com/Microsoft/UWPCommunityToolkit</projectUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>A library to integrate gaze interactions using eye trackers into UWP applications</description>
+    <releaseNotes>Preview Release</releaseNotes>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <tags>UWP Toolkit Windows Eye Gaze EyeTracker </tags>
+  </metadata>
+
+  <files>
+      <!-- WinMd and IntelliSense files -->
+      <file src="..\Microsoft.Toolkit.Uwp.Input.GazeInteraction\Output\Win32\Release\Microsoft.Toolkit.Uwp.Input.GazeInteraction.winmd" target="lib\uap10.0"/>
+      <file src="..\Microsoft.Toolkit.Uwp.Input.GazeInteraction\Output\Win32\Release\Microsoft.Toolkit.Uwp.Input.GazeInteraction.xml" target="lib\uap10.0"/>
+
+      <!-- DLLs and resources -->
+      <file src="..\Microsoft.Toolkit.Uwp.Input.GazeInteraction\Output\ARM\Release\Microsoft.Toolkit.Uwp.Input.GazeInteraction.dll" target="runtimes\win10-arm\native"/>
+      <file src="..\Microsoft.Toolkit.Uwp.Input.GazeInteraction\Output\ARM\Release\Microsoft.Toolkit.Uwp.Input.GazeInteraction.pri" target="runtimes\win10-arm\native"/>
```

Comment:
You should be able to put just a single .pri file in the \lib\ folder next to the winmd file